src/System.Private.CoreLib/src/Internal/Runtime/Augments/EnvironmentAugments.Unix.cs

```diff
@@ -37,5 +37,7 @@ private static void ExitRaw()
         {
             Interop.Sys.Exit(s_latchedExitCode);
         }
+
+        public static int ProcessorCount => Environment.ProcessorCount;
```

Comment:
It is better to add it to `EnvironmentAugments.cs` that is shared between Unix and Windows.